js/create.js

```diff
@@ -9,31 +10,56 @@ form.addEventListener('submit', event => {
   const inputForOptionalAnswer1 = form['question-box__answer']
   const inputForOptionalAnswer2 = form['question-box__answer1']
   const inputForOptionalAnswer3 = form['question-box__answer2']
-  const inputForCorrectAnswer1 = form['radio-button']
-  const inputForCorrectAnswer2 = form['radio-button1']
-  const inputForCorrectAnswer3 = form['radio-button2']
   const inputForHashtags = form['question-box__tags']
 
-  const quiz = [
-    {
-      question: inputForQuestion.value,
-      'optional-answers': {
-        a: inputForOptionalAnswer1.value,
-        b: inputForOptionalAnswer2.value,
-        c: inputForOptionalAnswer3.value,
-      },
-      answer: {
-        'a correct?': inputForCorrectAnswer1.checked,
-        'b correct?': inputForCorrectAnswer2.checked,
-        'c correct?': inputForCorrectAnswer3.checked,
-      },
-      hashtags: inputForHashtags.value,
+  const answerList = document.querySelectorAll(
```

Comment:
You can access the `radio` input via the `form` as well.

`form['radio-button']`

This will give you a NodeLIst of RadioButtons:

https://developer.mozilla.org/en-US/docs/Web/API/RadioNodeList

Reply:
If you give your `<input type="radio">` a value, you can access the value of the checked radio by doing:
`form['radio-button'].value`

You could give them the values `"a"`, `"b"` and `"c"` and that way I think you could simplify your code, removing the loop and the switch you have after that. Maybe take a look :)